document-parsers/langchain4j-document-parser-apache-tika/src/main/java/dev/langchain4j/data/document/parser/apache/tika/ApacheTikaDocumentParser.java

```diff
@@ -36,13 +38,15 @@ public class ApacheTikaDocumentParser implements DocumentParser {
     private final Supplier<Metadata> metadataSupplier;
     private final Supplier<ParseContext> parseContextSupplier;
 
+    private final boolean includeMetadata;
+
     /**
      * Creates an instance of an {@code ApacheTikaDocumentParser} with the default Tika components.
      * It uses {@link AutoDetectParser}, {@link BodyContentHandler} without write limit,
-     * empty {@link Metadata} and empty {@link ParseContext}.
+     * empty {@link Metadata} and empty {@link ParseContext}. By default, no metadata is included.
```

Comment:
@rzo1 , is it a typo? Am I read correctly, that `false` means NOT included? Then the javadoc comment is not correct

Reply:
`false` is similar to "do not include metadata in the parsed" document. 

The Supplier for `Metadata` via the ctr argument is a way to customize the actual metadata for a parsed document.

I will change the sentence to: "Note: By default, no metadata is added to the parsed document.". Hopefully, this makes it clearer?